lib/models/custom_layers/bus_stop_layer.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:async';
+import 'dart:developer';
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+
+import 'package:latlong/latlong.dart';
+
+import 'custom_layer.dart';
+
+class BusStopLayer extends CustomLayer {
+  List<Marker> markers = [];
+  BusStopLayer() : super("BusStopLayer") {
+    loop();
+  }
+  Future<void> loop() async {
+    refresh();
+    await Future.delayed(const Duration(seconds: 2));
+    log("loop");
```

Comment:
This part looks pretty hacky.

Reply:
the feature is still on progress it should be replaced by some fetch, the loop is just for simulate the possible behavior